store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/operation/RemoveAndDeleteGraph.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2017-2022 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.gchq.gaffer.federatedstore.operation;
+
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+
+@JsonPropertyOrder(value = {"class", "graphId"}, alphabetic = true)
+public class RemoveAndDeleteGraph extends RemoveGraph {
+    /*
+     * This class is completely empty.
+     * This highlights need for Operation.class to be final with field "String id = operationName"
+     */
```

Comment:
Can this be fixed with some other method? An empty class like this is not ideal.

Reply:
The comment signifies the Empty class is deliberate.
Do you want the "this highlights" part to be removed?